src/base/_component.scss

```
@@ -18,6 +18,17 @@
 .bg-norepeat {
 	background-repeat: no-repeat;
 }
+.bg-half-width-right,
+.bg-half-width-left {
```

I propose the shorter name

```suggestion
.bg-half-right,
.bg-half-left {
```